packages/react/src/components/LocaleModal/LocaleModalCountries.js

```diff
@@ -208,6 +212,21 @@ export const filterLocale = (
     localeItems.length === localeItemsHidden.length
       ? modalLabels.unavailabilityText
       : modalLabels.availabilityText;
+
+  /**
+   *  Reflect number of results in the accessibility readout
+   */
+  const resultsElement = document.querySelector(
+    `.${prefix}--locale-modal__list .${prefix}--assistive-text`
+  );
```

Comment:
Thank you for the update @IgnacioBecerra, another step; Would you try getting `ref` of the top-level `<div>` and run `.querySelector()` from there?